app-quizz/src/app/step-6/page.tsx

```diff
@@ -1,29 +1,14 @@
 "use client";
 
 import React from "react";
-
-const options = [
-  {
-    id: 1,
-    title: "What is Scope?",
-    description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum",
-  },
-  {
-    id: 2,
-    title: "What is Closure?",
-    description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum",
-  },
-  {
-    id: 3,
-    title: "What is Scope?",
-    description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum",
-  },
-];
+import renderBoxOptions from "../utils/renderBoxOptions";
+import options from "../data/responses";
+import Button from "../components/Button/button";
 
 export default function StepSix() {
+  const bodyContainerStyles =
+    "md:container md:mx-auto min-h-full ml-4 mr-4 pt-32 bg-teal-100 pb-20";
+
   const handleClick = (option: number) => {
```

Comment:
S: I'd like to suggest renaming this `handleClick` to handlerClick. What do you think?